app/sequencescape_excel/sequencescape_excel/helpers/worksheet.rb

```diff
@@ -11,8 +11,7 @@ module Worksheet
           border: {
             style: :thin,
             color: '00'
-          },
-          format_code: '@'
```

Comment:
I've remove this as the default type, and now it only applies to 'string' columns.

Reply:
What does the @ do?

Reply:
Marks it as a text field. I should probably add a comment, as excel format_codes are somewhat impeneterable